### PR TITLE
Consolidate Terminal Styles

### LINKED ISSUE #611: Runme terminal settings
Runme Terminal, should allow more customization, so far we support Font family and Font size, and there are a bunch of terminal settings that can be provided like:

- Scrollback: Amount of retained rows when lines are scrolled beyond the initial viewport
- Fast scroll sensitivity: The scroll speed multiplier used for fast scrolling
-  Cursor style
- Cursor width
Review what other settings can be supported

### PR DESCRIPTION
**Pull request by christian-bromann**
Fixes #611

### CODE REVIEW

**Comment by christian-bromann** [comment]
This is now ready for review.

File: package.json [hunk 1]
```diff
@@ -453,6 +453,34 @@
...
+          "runme.terminal.cursorWidth": {
+            "type": "number",
+            "markdownDescription": "The width of the cursor in CSS pixels when `cursorStyle` is set to 'bar'."
+          },
+          "runme.terminal.smoothScrollDuration": {
+            "type": "number",
+            "default": 0,
+            "markdownDescription": "The duration to smoothly scroll between the origin and the target in milliseconds. Set to 0 to disable smooth scrolling and scroll instantly."
+          },
+          "runme.terminal.scrollback": {
```
Comment (degrammer) [inline package.json:480]: I think is better to have a default value of 100, see integrated terminal settings for VS Code

**Review by degrammer — APPROVED** [review]
LGTM, I've added some suggestion to the default scrollback setting, also, for a next iteration, I imagine the cursor width setting with something like this:
Screenshot 2023-09-13 at 9 59 51 AM

File: package.json (hunk 1 above)
Comment (christian-bromann) [reply package.json:480]: Currently the implemented behavior mimics what we have in `main` which is not setting the value at all and have it default to the [default value](https://github.com/xtermjs/xterm.js/blob/bdb25c2dbee65af8afa36dca816af51fe5ab1d5d/src/common/services/OptionsService.ts#L33). Are you suggesting to explicitly set the default value to `100`?

**Comment by christian-bromann** [comment]
> I imagine the cursor width setting with something like this

As mentioned above: the current behavior will not set the value when undefined, so we rely on the xTerm default value.

**Comment by christian-bromann** [comment]
> I imagine the cursor width setting with something like this

As mentioned above: the current behavior will not set the value when undefined, so we rely on the xTerm default value.

**Pull request merged**